docker-compose-published.yaml

```diff
@@ -0,0 +1,235 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/compose-spec/compose-spec/master/schema/compose-spec.json
+x-common-environment: &common-environment
+  FREQUENCY_API_WS_URL: ${FREQUENCY_API_WS_URL:-wss://0.rpc.testnet.amplica.io4}
+  SIWF_NODE_RPC_URL: ${SIWF_NODE_RPC_URL:-https://0.rpc.testnet.amplica.io4}
```

Comment:
nit: defaulting to testenet or local?

Reply:
I'll fix the typo and keep it for testnet, but the env vars takes precedence and start.sh should set them.